smartsim/_core/control/controller.py

```diff
@@ -110,6 +160,10 @@ def start(
         self._jobs.kill_on_interrupt = kill_on_interrupt
         # register custom signal handler for ^C (SIGINT)
         signal.signal(signal.SIGINT, self._jobs.signal_interrupt)
+
+        # have job manager launch a telemetry monitor if any jobs are run
+        self._jobs.add_job_onstart_callback(create_telmon_callback(exp_path))
```

Comment:
Great catch. the controller is used to automatically start the telemetry monitor after it shuts itself down and doesn't have any other onstart callbacks for the job manager.

It may not work if we add more callbacks, but for now I will ensure it isn't duped by checking if the controllers job manager already has an onstart callback.